packages/client/mocha.opts

```diff
@@ -1,7 +0,0 @@
---require ts-node/register
```

Comment:
With or even without this file, I cannot run the tests from the launch.json `Run current test` or `Run all tests`. Even before any execution I get:
```
✖ ERROR: --opts: configuring Mocha via 'mocha.opts' is DEPRECATED and no longer supported.
          Please use a configuration file instead.         
```

So it would be great if you could either remove the two entries from launch.json since the file is no longer there anyway or update the calls accordingly.

Reply:
Fixed